libexec/start-target

```diff
@@ -37,6 +37,10 @@ case $VMSW in
     echo "Gitian-${2}" > var/target.vmname
     ;;
   DOCKER)
-    docker run -d --name gitian-target base-$SUFFIX:latest > /dev/null
+    EXTRA_ARGS=""
+    if [ -n "$ALLOW_PRIVILEGED" ]; then
```

Comment:
There's a slight security issue here.  The user might have `ALLOW_PRIVILEGED` env var set for some other binary in a different package - i.e. because of a env var naming collision.  This will unexpectedly enable privileged mode here and potentially open the user to an attack via a compromised docker image.

So I think we should either conclusively set the var in `gbuild`, or prefix the name with `GITIAN_` to ensure there's no collision in normal use.

Reply:
Changed to `GITIAN_ALLOW_PRIVILEGED`.